kombuworker/agnostic.py

```diff
@@ -25,7 +28,11 @@ def parse_queue(
 
 
 def insert_task(
-    queue_url: str, tool_name: str, *args: list, queue_name: str = None, **kwargs: dict,
+    queue_url: str,
+    tool_name: str,
+    *args: list,
+    queue_name: str = None,
+    **kwargs: dict,
```

Comment:
This repo is black formatted, which keeps the old format for these changes.

Reply:
I didn't mean to make any of the style changes -- I just used `black==21.9b0`.
I know different black versions act differently, which is probably the source of the misunderstanding.
Which version did you use?

Reply:
Yup, good catch. I'm using an older version (19.10b0). I can make a new commit once we merge this to update the black version.